### PR TITLE
Cthub 147 - Backend for creating application 

### PR DESCRIPTION
**Pull request by emi-hi**
-rushing stats stuff has been added back (was deleted recently) for testing purposes, once things are working i'll delete it all
-currently giving CORS error

 has been blocked by CORS policy: Response to preflight request doesn't pass access control check: The value of the 'Access-Control-Allow-Credentials' header in the response is '' which must be 'true' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.

### CODE REVIEW

**Comment by naomiaro** [comment]
I had some luck with figuring out the config to be able to keep going with building this. I put my work in this branch: https://github.com/naomiaro/itvr/tree/feat/first-form

You could set my fork as a remote and pull its contents into a branch in your repo if you'd like to check out some differences.

File: docker.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Example to use Docker instead of containerd & nerdctl
```
Comment (naomiaro) [inline docker.yaml:1]: we should get rid of this file since it's just for lima config.

File: django/api/urls.py [hunk 2]
```diff
@@ -13,13 +12,20 @@
...
 
+ROUTER.register(
+    r'application-form?$', ApplicationFormViewset, basename='application-form'
+)
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include(ROUTER.urls)),
-]
+    re_path('/', TemplateView.as_view(template_name='index.html')),
+]
```
Comment (naomiaro) [inline django/api/urls.py:31]: missing a newline here

File: django/api/serializers/application_form.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            date_of_birth=details.get('dob'),
+            tax_year=details.get('tax_year'),            
+        )
+        return obj
+
+
+class ApplicationFormSerializer(ModelSerializer):
+    class Meta:
+        model = IncomeVerification
+        fields = '__all__'
```
Comment (naomiaro) [inline django/api/serializers/application_form.py:25]: missing a newline here

File: react/src/rebate_application_form/ApplicationFormContainer.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      [name]: value,
+    });
+  };
+  const handleSubmit = () => {
+    axios.post(ROUTES_APPLICATION.SAVE, { details })
+      .then((response) => { console.log(response); });
+  };
+  useEffect(() => {
+    setLoading(true);
+    // do an axios get for whatever informaion we need to retrieve for the form to be filled
```
Comment (naomiaro) [inline react/src/rebate_application_form/ApplicationFormContainer.js:25]: Do we need this effect?

File: react/src/rebate_application_form/routes.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+const API_BASE_PATH = "/api/application-form";
+
+const APPLICATION = {
+  SAVE: `${API_BASE_PATH}`,
+  LIST: "/api/application-form",
```
Comment (naomiaro) [inline react/src/rebate_application_form/routes.js:5]: Can use API_BASE_PATH here as well.

**Review by naomiaro — CHANGES_REQUESTED** [review]
Wrote a couple comments on the pr

File: react/src/rebate_application_form/ApplicationFormContainer.js (hunk 4 above)
Comment (emi-hi) [reply react/src/rebate_application_form/ApplicationFormContainer.js:25]: no, i thought we might be retrieving data for the form but i'll take it out until we do

**Review by naomiaro — APPROVED** [review]

**Pull request merged**